lib/src/HttpControllersRouter.cc

```diff
@@ -490,7 +490,7 @@ void HttpControllersRouter::route(
         for (auto &item : ctrlVector_)
         {
             auto const &ctrlRegex = item.regex_;
-            if (std::regex_match(req->path(), result, ctrlRegex))
+            if (std::regex_match(req->path(), result, ctrlRegex) && item.binders_[req->method()])
```

Comment:
I think the `item.binders_[req->method()]` should be checked first.

Reply:
Yeah, you are right, sorry :sweat: